nightingale_dispatcher/src/nightingale_dispatcher/main.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import rospy
+
+import queue
+from nightingale_msgs.srv import AddTaskDispatcher, CancelTaskDispatcher
+from nightingale_dispatcher.subtask import Subtask
```

Comment:
So tasks(functions which actually execute behaviour)  are called "subtasks"? Why not call them tasks?